backend/src/Utils/Datatypes.hs

```diff
@@ -28,6 +36,9 @@ data Occupancy = Occupancy { oTitle :: String -- ^ The title of the event, inten
                            , oEnd   :: UTCTime -- ^ When does this occupancy end?
                            } deriving (Show)
 
+-- | Some temporary identifier for a user
+type UserId = Int
```

Comment:
perhaps make this a `newtype` to prevent mistakes

Reply:
I don't think we should do that, we want it to behave like an integer for easy serialization to the frontend. It really doesn't matter if you use an int. Casting only introduces more complexity.